### PR TITLE
Bug 560433 - [Passage] add org.eclipse.passage.loc.workbench.tests

### PR DESCRIPTION
**Pull request by ruspl-afed**
Added org.eclipse.passage.loc.workbench.tests bundle
Added ClassifierMetadataTest

Signed-off-by: Alexander Fedorov <alexander.fedorov@arsysop.ru>

### CODE REVIEW

File: bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/SelectRequest.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import org.eclipse.passage.loc.jface.dialogs.Appearance;
+
+/**
+ * Encapsulates the information required to select the classifier
+ * 
+ * @since 0.6
+ *
+ * @param <R> target type to be selected
+ */
+public final class SelectRequest<R> {
```
Comment (eparovyshnaya) [inline bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/SelectRequest.java]: Cool class, the only thing bothers me about it is how it is connected to the MR dedication?

File: bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/SelectRequest.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	 *                   <code>null</code>
+	 * @param input      the input to select from, must not be <code>null</code>
+	 * @param appearance the appearance of UI dialog, must not be <code>null</code>
+	 */
+	public SelectRequest(Class<R> target, String domain, Supplier<Iterable<R>> input, Appearance appearance) {
+		this.target = target;
+		this.domain = domain;
+		this.input = input;
+		this.appearance = appearance;
+	}
```
Comment (eparovyshnaya) [inline bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/SelectRequest.java]: If these guys are demanded to be not *null*, can we add `Objects.requireNotNull` checks?

File: bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/SelectRequest.java (hunk 1 above)
Comment (ruspl-afed) [reply bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/SelectRequest.java]: It appeared here unintentionally, I will cover it with the test as well

File: bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/SelectRequest.java (hunk 2 above)
Comment (ruspl-afed) [reply bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/SelectRequest.java]: Yes, sure

**Pull request merged**